build/repo.targets

```diff
@@ -64,10 +64,7 @@
         <Category>noship</Category>
       </ArtifactInfo>
 
-      <FilesToSign Include="$(AncmZipOutputPath)" IsContainer="true" />
-
-      <FilesToSign Include="@(Components->'%(BaseOutputPath)/%(Platform)/%(NativeAsset).dll')" Container="$(AncmZipOutputPath)" Certificate="$(AssemblySigningCertName)" />
-
+      <FilesToExcludeFromSigning Include="$(AncmZipOutputPath)" IsContainer="true" />
```

Comment:
Maybe I'm reading this wrong, but wasn't this including AncmZipOutputPath before and now its excluding?

Reply:
No, you're reading it right. I typed the wrong thing. Will fix it up.